spec/debug.spec.js

```diff
@@ -5,21 +5,76 @@ const runner = require('./runner');
 describe('debug.spec: debug namespace tests', () => {
     describe('debug.log', () => {
         it('logs a simple message', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.debug.log('Hello! This is a simple message');
                 let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
                 await __close(logFileContent);
             `);
             assert.ok(runner.getOutput().includes('Hello! This is a simple message'));
         });
 
+        it('logs an info message', async () => {
+            runner.run(`
+                await Neutralino.debug.log('Hello! This is a info message', 'INFO');
+                let logFileContent = await Neutralino.filesystem.readFile(NL_PATH + '/neutralinojs.log');
+                await __close(logFileContent);
+            `);
+            assert.ok(runner.getOutput().includes('Hello! This is a info message'));
```

Comment:
The `INFO` option outputs the `INFO` word into the logfile, so can we add another assertion for it as well?